frontend/docs/seo-guide.md

```diff
@@ -0,0 +1,393 @@
+  - **Provide a 403 error page** (access denied).
+  - **Provide a 404 error page** (Page not found).
+- **5XX:** These are all server error messages. Ensure your page does not return 500 errors.
+
+#### Sitemap
+
+I should start with this, Google does not index your pages because you asked for it. Google indexes pages because it founds them and considers them good enough quality to be indexed. 
+
+Sitemaps are machine readable URL submission lists in various formats, e.g., XML or plain text. It’s like an archive of every webpage in your website. **Use your sitemaps as sleuthing tools to discover and eliminate indexation problems**, and only let/ask Google to index the pages you know Google is going to want to index.
+
+Sitemaps are part of Robots Exclusion Protocol (REP), which we will cover in some lines.
+
+- If you’ve got a big site, use dynamic sitemaps. You can use plugins for this, for example: [DynamicSitemaps (RoR)](https://github.com/lassebunk/dynamic_sitemaps).
+- Not include pages in your sitemap that are blocked by robots.txt or have set meta robots `'noindex,follow'`.
+
+ [Read more...](https://moz.com/blog/xml-sitemaps)
+
+#### Canonical Tag
+Canonical URL **allows you to tell search engines that certain similar URLs are actually one and the same**. Sometimes you have products or content that is accessible under multiple URLs, or even on multiple websites. Using a canonical URL (an HTML link tag with attribute rel=canonical) these can exist without harming your rankings.
+
+You should use www or not in your web site’s canonical hostname? For SEO **it really doesn’t matter at all as long as you choose one**. For example:
+
+- https://www.my-page.com
+
+- https://my-page.com
+
+What do tou have to do the set the canonical URL?
+
+1. First, you pick one of your two pages as the canonical version. It should be the version you think is the most important one.
+2.  Add a `rel=canonical` link from the non-canonical page to the canonical one in the `<head>` section.
+
+```html
+<link rel="canonical" href="https://www.my-page.com/">
+```
+
+_Why:_
+
+> This method help us to prevent problems caused by identical or "duplicate" content appearing on multiple URLs. [Read more...](https://moz.com/learn/seo/canonicalization)
+
+#### URLs
+
+Search engines and visitors can interact with your website more effectively when you implement URLs that describe your pages content.
+
+  - URLs should not be grater than 100 charaters.
+  - Avoid use more than 3 levels in your URLs. What I mean with level? https://yourdomain.com/level1/level2/level3.
+  - URLs must be user-friendly. An URL have to describe page content and include keyword in it. For example:
+    - BAD: https://yourdomain.com/123123123123lkjlkjlkjlkj.
+    - GOOD: https://yourdomain.com/how-to-implement-good-seo-strategy.
+- There are no params in the URL, this can be setup in Google Search Console.			
```

Comment:
this is not indented and i don't understand what you mean here

Reply:
I think I fixed the indentation. 
About text meaning, what I tried to explain that URLs must have your best keywords and human-readable text in it. You should avoid ids or not-friendly text in them, like the examples below